### ISSUE TITLE
Can't update flash card after I edit obsidian and run update command

### ISSUE THREAD

**Issue by Jackiexiao** [reuseman/flashcards-obsidian#2]
this is my content
```
### how to xxx #card
something
^1607617939360
```

**To Reproduce**
- Steps to reproduce the behavior:
1. run command: Flashcards : Generate for the current file
2. view in anki
3. change obsidian text
4. run command again
5. view in anki, nothing change

BTW, is there any command : `Generate for all the file`?

**Comment by reuseman** [comment]
Hey!

Probably you just came across a bug of the APIs that I use to communicate with Anki. And the bug is that when you try to update and the **Browse** window of Anki is opened the update does not work properly. So try to close it and let me know. I need to insert this in the documentation.

About the `Generate for all the file`, it will arrive. But before I prefer to push the other update that is arriving this evening, with a bunch of improvements about the obsidian scheme support and the update of global tags

**Comment by Jackiexiao** [comment]
it add new card instead of update the original card

**Comment by Jackiexiao** [comment]
and if I have new note in obsidian, I have to delete the yaml config at the top of markdown file, only after that, new card will add into anki (sometimes it work, sometimes I need to run command several time
```
---
cards-deck: Default
---
```

**Comment by reuseman** [comment]
> it add new card instead of update the original card

The text you wrote is totally correct and is parsed correctly. The only explanation is that it doesn't read the `^1607617939360`. Is that on a new line compared to the content when you edit it?

> and if I have new note in obsidian, I have to delete the yaml config at the top of markdown file, only after that, new card will add into anki (sometimes it work, sometimes I need to run command several time

So you create a new empty note, then you insert the `cards-deck: Default`, then you write the content of the card, and when you run the command it does not work? And when you delete, and it does work, the yaml config is added automatically right?
`

**Comment by reuseman** [comment]
@Jackiexiao now it should be fixed. Let me know

**Comment by Jackiexiao** [comment]
I have been busy these day, check later! thx for developing this great plugin!

**Comment by Jackiexiao** [comment]
works well, thx!
